Data Structures/Array/dynamic_array.cpp

```diff
@@ -0,0 +1,86 @@
+//Name: Kumar Raju.B
+//E-mail: bim2017008@iiita.ac.in
+//TITLE: DYNAMIC ARRAY-VECTOR(STL)
+#include<bits/stdc++.h>
+using namespace std;
+void displayArray( vector<int> v)
+{
+    vector<int> :: iterator it;
+    for( it = v.begin() ; it!= v.end() ; ++it)
+        cout<<*it<<" ";
+    cout<<endl;
+}
+int main()
+{     int i, n, value;
+      vector<int> array;     
+      cout<<"Input the number of elemnts you want to store in array:\n";
+      cin>>n;
+      cout<<"Enter the elements:\n";
+      for( i = 0 ; i < n; ++i)
+      {
+          cin>>value;
+          array.push_back(value);
+      }
+
+      // Syntax to append a value at the end of the array ,time complexity:O(1)
+      array.push_back(10); 
+      
+      array[3] = 7; // Syntax to modify a value of the array
+      
+      // To find the number of elements in the array ,time complexity:O(1)
+      cout<<"The size of the array is : "<<array.size()<<endl; 
+      
+      //The format to know the value at particular index is :(for ex. at 4) ,time complexity:O(1)
+      cout<<"The value at index 4 is "<<array[4]<<endl;
```

Comment:
What if the user enters array size less than 5? This will be a garbage value. Fix it.